adagrams/game.py

```diff
@@ -1,11 +1,89 @@
+import random
+
+
+
 def draw_letters():
-    pass
+    letters = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+    relative_weights = [9, 2, 2, 4, 12, 2, 3, 2, 9, 1, 1, 4, 2, 6, 8, 2, 1, 6, 4, 6, 4, 2, 2, 1, 2, 1] #how many of each letter in pool of letters
```

Comment:
Index-matching is a little fragile. If you instead represent this as a list of tuples or as a dictionary, it's more obvious if something is wrong or missing. For example, I made a mistake when I modified your code, below. What did I mess up? Was it obvious?

```suggestion
    letters = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "ξ", "S", "T", "U", "V", "W", "X", "Y", "Z"]
    relative_weights = [9, 2, 2, 4, 12, 2, 3, 2, 9, 1, 1, 4, 2, 6, 8, 2, 1, 8, 6, 4, 6, 4, 2, 2, 1, 2, 1] #how many of each letter in pool of letters
```